client/analysis/output.go

```diff
@@ -37,19 +37,19 @@ func CheckContainerOutput(container types.Container) {
 }
 
 // PrintGosecOutput will print the Gosec output.
-func PrintGosecOutput(containerOutput string) {
```

Comment:
Actually client is not analysing the `containerOutput` itself. It is viewing mongoDB field, that was already been handled by huskyCI API.